assets/stylesheets/common/wizard-admin.scss

```diff
@@ -707,3 +707,26 @@
   width: 80px;
   vertical-align: middle;
 }
+
+.btn-pavilion-pro {
+  background: #3c1c8c;
+  color: white;
+
+  .d-icon {
+    color: white;
+  }
+
+  &:hover,
+  &:focus {
+    background: #2d1569 !important;
```

Comment:
@keegangeorge We want to avoid using ``!important``. Also, let's use variables here too. Try not to use hex codes directly in the CSS.